src/deepsparse/transformers/haystack/nodes.py

```diff
@@ -192,22 +193,22 @@ def __init__(
         emb_extraction_layer: Union[int, str, None] = -1,
         pooling_strategy: str = "cls_token",
         top_k: int = 10,
-        embed_title: bool = False,
         progress_bar: bool = False,
         scale_score: bool = True,
         context: Optional[Context] = None,
+        title_separator: Optional[str] = " ",
```

Comment:
Do you envision people actually changing the separator or is this mainly just a yes/no for including?

Reply:
Main use cases would be a space " " or a "<SEP>" token (I've confirmed it gets tokenized properly)

Reply:
My 2 cents are the less configuration options ppl have to understand the better. So i'd lean towards just picking one of those two and going back to the embed_title parameter.

Ofc if the title separator option is commonly done within haystack then go with that!

Reply:
Ha that's fair enough. I'll go back to embed_title (IIRC Haystack implements it the same way with a space)